pkg/getty/etc/profile.d/namespace.sh

```diff
@@ -1 +1,23 @@
-export PS1="(ns: getty) $PS1"
+#/bin/sh
+
+in_a_ns() {
+	IN_A_NS=0
+	NOT_IN_A_NS=1
+
+	# If PID 1 != /sbin/init then we are in a container
+	if ! pgrep /sbin/init > /dev/null; then
+		return $IN_A_NS
+	fi
```

Comment:
This checks whether there is any process in the current pid namespace named `/sbin/init` but doesn't check if it is pid 1 AFAICT. An alternative to pgrep would be to consult `/proc/1/cmdline`.